frontend/src/components/DirectMessage/Messaging.js

```diff
@@ -281,7 +337,7 @@ const Message = () => {
             <div style={{ overflowY: "scroll", height: "500px" }}>
               {/* Maps through all the users in the 'users_information' array, 
               except the current user, and renders a 'user-tab' div for each */}
-              {users_information.map(
+              {connections.map(
```

Comment:
The feature works, and I can only see the DMs with people I'm connected with. Nice work!